tests/Phpixela/ApiTest.php

```diff
@@ -0,0 +1,245 @@
+<?php
```

Comment:
I could not apply this patch at this time.

Could you fix code indentation, naming and something like existing codes?